### DISCUSSION TITLE
Times via Due:

### DISCUSSION THREAD

**Discussion by redrabbitrob** [giovannicoppola/AlfreDo#15] [General]
Hey  :),

Is there any possibility to add the time after you choose one of the options to enter a date, like

"!!! due: - > due:0d12:00 Homework " 

For example? 
Or is there any option, and I'm doing it wrong? Thanks for your help. 

Cheers Rob

**Comment by giovannicoppola** [comment]
you can add time with this format `YYYY-MM-DDTHH:MM`, but not after the shortcuts (`2d` etc). 
It's a good idea, I will add it to the roadmap!

**Reply by redrabbitrob** [reply]
This would be so awesome... just to type the shortcut and then, for example, the time of day right afterward... ;)fast and simple

**Comment by giovannicoppola** [comment]
@redrabbitrob this is now added in [version 0.4](https://github.com/giovannicoppola/AlfreDo/releases/tag/v0.4). Let me know if you see anything off!
